src/UI/Core/Window.cpp

```diff
@@ -12,7 +12,6 @@
 
 using namespace NAS2D;
 
-const float WINDOW_TITLE_BAR_HEIGHT = 20.0f;
```

Comment:
I see this constant has been moved to the header file. I would recommend placing it in the `protected` section. That would be a simpler change, and would not require adding a new `protected` `static` method for `getWindowTitleBarHeight`. Either which way, a new `protected` member must be added, either as a constant, or as a static function. Given the definition of the new static function, it's unlikely it would ever return anything other than a constant anyway.